Content.Shared/Blocking/BlockingSystem.cs

```diff
@@ -184,20 +197,24 @@ public bool StartBlocking(EntityUid item, BlockingComponent component, EntityUid
                 }
             }
 
-            //Don't allow someone to block if they're somehow not anchored.
-            _transformSystem.AnchorEntity(user, xform);
-            if (!xform.Anchored)
-            {
-                CantBlockError(user);
-                return false;
-            }
+            // //Don't allow someone to block if they're somehow not anchored.
+            // _transformSystem.AnchorEntity(user, xform);
+            // if (!xform.Anchored)
+            // {
+            //     CantBlockError(user);
+            //     return false;
+            // }
             _actionsSystem.SetToggled(component.BlockingToggleAction, true);
             _popupSystem.PopupEntity(msgUser, user, user);
             _popupSystem.PopupEntity(msgOther, user, Filter.PvsExcept(user), true);
         }
 
         if (TryComp<PhysicsComponent>(user, out var physicsComponent))
         {
+            // Makes the user able to push things around with their body and slows them greatly
+            _physics.SetBodyType(user, BodyType.Dynamic);
```

Comment:
Players should never be dynamic body type, if you want them pushing other entities around it needs implementing as otherwise this will cause many physics bugs on content (e.g. it's trivial to push mobs through walls now).